presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -15,44 +28,63 @@ The driver is also available from Maven Central:
         <version>\ |version|\ </version>
     </dependency>
 
-Requirements
-------------
+You should be using the same version of the JDBC driver as your Presto
+installation. If you are not on the latest release shown above, a list of all
+currently available versions can be found `here <https://repo1.maven.org/maven2/io/prestosql/presto-jdbc>`__.
 
-The JDBC driver is compatible with Java 8, and can be used with applications
-running on Java virtual machines version 8 and higher.
+Once downloaded, you must add the JAR file to the classpath of your Java 
+application. Often, this will be the lib folder of a Java application, but
+you should check the documentation for your specific application. Depending 
+on your application, you may be able to accomplish this from a dialog rather 
+than on the command line.  
 
-Driver Name
------------
+Once you have downloaded the JDBC driver and added it to your 
+classpath, you'll typically need to restart your application in order to 
+recognize the new driver. After that, depending on your application, you 
+may need to manually register and configure the driver.
 
-The driver class name is ``io.prestosql.jdbc.PrestoDriver``.
-Most users do not need this information as drivers are loaded automatically.
+Registering & configuring the driver
+------------------------------------
 
-Connecting
-----------
+Drivers are commonly automatically loaded by applications once they are added 
+to its classpath. If your application does not, this section will be helpful.
 
-The following JDBC URL formats are supported:
+If your application has a UI, look for a Settings option or a Drivers tab and 
+register the JDBC driver there. The steps to register the JDBC
```

Comment:
How many (%) applications have "Drivers" tab in "Settings"?

Reply:
Using the definitive guide as a model, touching on this briefly here.